storybook/stories/online/README.md

```diff
@@ -5,15 +5,16 @@ The Online hook listens for browser online/offline events to determine if there
 Import as follows:
 
 ```javascript
-import { useOnline } from '@nearform/react-browser-hooks' 
+import { useOnline } from '@nearform/react-browser-hooks'
 ```
 
 Example of usage:
 
 ```javascript
-const status = useOnline()
-<p>Status: {status.online ? 'online' : 'offline'}<br/>
+const online = useOnline()
```

Comment:
The documents are now aligned with the existing implementation. I think @jh3y 's suggestion was to change to hook to return an object with an `online` property instead of status. What do you think about?

```
const { online } = useOnline()
```

Reply:
I wouldn't even return an Object. No need, it's always a Boolean value 👍

Reply:
I see. Yeah that makes sense. I'll update

Reply:
I think we originally decided  to keep this as an object for consistency with the other hooks, as most return extra state?

Reply:
The idea was to not modify anything from the native APIs. 

So if you listened for an orientation change, a geolocation event for example, you get the untouched object back in state.

But there's no need to wrap this in an Object. It's not done for the visibility hook.

Reply:
Ok, cool. Agreed!